### PR TITLE
Issue#8482

### LINKED ISSUE #8482: Add test case for QueryResponseRow.getData

### PR DESCRIPTION
**Pull request by huanghao495430759**
Fixes #8482 .

### CODE REVIEW

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/response/data/QueryResponseRowTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import java.util.Collections;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class QueryResponseRowTest {
```
Comment (tristaZero) [inline shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/response/data/QueryResponseRowTest.java:34]: It's better to define it as a `final` class if there is no specific intent.

**Review by tristaZero — CHANGES_REQUESTED** [review]
@huanghao495430759 Hi Glad to see your another PR here. :)
It is supposed to be merged after you fix some small tips, IMO.

File: shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/response/data/QueryResponseRowTest.java (hunk 1 above)
Comment (huanghao495430759) [reply shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/response/data/QueryResponseRowTest.java]: OK

**Review by tristaZero — APPROVED** [review]
@huanghao495430759 Nicely done. :)

**Pull request merged**